functions/strategies/FirefoxStrategy.js

```diff
@@ -1,5 +1,22 @@
-const firefoxStrategy = (req, _store) => new Promise((resolve, reject) => {
 
+const firefoxWithProxy = (req, store) => new Promise((resolve, reject) => {
+  const eventListener = (event) => {
+    window.removeEventListener("firefoxExtSendRequestComplete", event);
+
+    if (event.detail.error) reject(JSON.parse(event.detail.error));
+    else resolve(JSON.parse(event.detail.response));
```

Comment:
```js
    if (event.detail.error) {
      reject(JSON.parse(event.detail.error));
    } else {
      resolve(JSON.parse(event.detail.response));
     }
```
```

Reply:
You haven't made this change yet!

Reply:
It's such a tiny detail and it is like 4AM here, soooooo, sleep deprivation just kicked in sorry :stuck_out_tongue: